tests/src/psa_crypto_helpers.c

```diff
@@ -128,4 +128,22 @@ psa_key_usage_t mbedtls_test_update_key_usage_flags( psa_key_usage_t usage_flags
     return( updated_usage );
 }
 
+/** Check that no PSA Crypto key slots are in use.
```

Comment:
The description is duplicated with that in head file, should we remove them?

Reply:
Right, it's removed.